test/fuzzer/pedro/index_current_timestamp.test

```diff
@@ -13,38 +13,31 @@ INSERT INTO t1(c1) VALUES(1);
 statement ok
 CREATE SEQUENCE seq
 
-statement error
-CREATE INDEX i1 ON t1((get_current_timestamp()), c1);
-----
-Cannot use get_current_timestamp
+#statement error
+#CREATE INDEX i1 ON t1((get_current_timestamp()), c1);
 
 statement error
 CREATE INDEX i1 ON t1((random()), c1);
-----
-Cannot use random
 
 statement error
 CREATE INDEX i1 ON t1((nextval('seq')), c1);
-----
-Cannot use nextval
 
 statement ok
 DELETE FROM t1;
 
 # now with an empty table
-# we can create the index
-statement ok
+
+statement error
 CREATE INDEX i1 ON t1((get_current_timestamp()), c1);
 
 # but attempting to insert a value borks it up
-statement error
+statement ok
 INSERT INTO t1 VALUES (42, 'hello')
-----
-Cannot use get_current_timestamp
 
 restart
 
 statement error
+CREATE INDEX i1 ON t1((get_current_timestamp()), c1);
```

Comment:
nit: I believe there is a way now to check the error returned. Maybe you have this in another PR though?

Reply:
Oh that is very neat to know, thanks!